### DISCUSSION TITLE
Mute all command

### DISCUSSION THREAD

**Discussion by PetyXbron** [EssentialsX/Essentials#5117] [Ideas and Feature Suggestions]
Hey guys,
the feature of **muting all players on the server** is missing in Essentials. I cannot use `/mute *` (maybe because the argument accepts offline players too) so I guess it's not feasible yet.

It would be a new command like `/muteall` or `/mutechat` and it would basically **block everyone from typing normal messages to chat** (so it doesn't apply to commands). Also, there should be permission to bypass this state with OPs too.

It's a good feature for **server staff broadcasting and telling something** to the online players and be not disturbed by other players and also keeps the messages in sight to get them easily accessible for players in the chat.
I think a lot of you already know this feature and I think it's an essential feature.

Peace

**Comment by pop4959** [comment]
ESC -> Options -> Chat Settings -> Chat: Hidden (or Commands Only if you need that)

**Reply by PetyXbron** [reply]
What? Are you kidding me? **This is an individual solution for clients only**.
If I want to broadcast something to all players and players should listen to me without disturbing the chat with more messages I can't hide my chat. Also, hiding chat like that stops me from chatting. I am sorry but you did not understand my suggestion.

Also, the `/mute` command is for admin purposes and for muting one player for everybody. It doesn't mute him for me only. **I want to mute every non-OP player** (or those, who don't have bypass permission) **to all players** to have an overview in the chat.

**Reply by pop4959** [reply]
Oh, you meant the other way around. 😀 

Essentials does have a /broadcast command which highlights a message in chat. This is what people normally use, since it doesn't block chat but stands out compared to a normal chat message.

If that doesn't work, can you explain your use case? Because blocking chat for everyone doesn't sound like it would be too useful on most servers (only those with a huge number of players), and it probably also wouldn't block out messages from other plugins or commands.

**Reply by PetyXbron** [reply]
Hi @pop4959, sorry for being a bit aggressive when replying to you.
I just remembered about this feature request. If you are interested in the **use case**, I will try to demo it to you.

So whenever we are playing a **special event** on our server I would like to **pause all chat messages from non-OP players**, because I want to **point out my (admin/OP) messages**, not just by /broadcast command but just **making it clearer**. And whenever the event is done **I could unmute everyone again**.

It's something like lockdown chat used on Discord. And I am not alone with this idea. **What do you think about this idea?**

**Reply by pop4959** [reply]
Thanks for responding! I think this is still very different from a "mute all" command and it would not be part of muting at all. I'm still not sure about the idea but if the rest of the team thinks this is a good idea we'll consider it.

**Reply by PetyXbron** [reply]
Hi! Thanks for responding too! **What can I do more** to push this suggestion to the new release at this point? Have you been able to hand over this idea further?
